app/routes/bdb/components/AddSemester.css

```diff
@@ -0,0 +1,16 @@
+.yearForm {
+  width: 100px;
```

Comment:
Isn't this width a little superflous with display:block? And are theses classes used? Couldn't seem to find any reference to styles.yearForm and styles.choices in AddSemester.tsx

Reply:
I just moved this over from `bdb.css` and yes they are used on line [129](https://github.com/webkom/lego-webapp/blob/ivarnakken/aba-720-update-bdb-colors/app/routes/bdb/components/AddSemester.tsx#L129) and [132](https://github.com/webkom/lego-webapp/blob/ivarnakken/aba-720-update-bdb-colors/app/routes/bdb/components/AddSemester.tsx#L132). The width is needed in the sense that without it it will cover up 100% of the page.